event_routing_backends/backends/tests/test_events_router.py

```diff
@@ -841,6 +854,7 @@ def test_successful_routing_of_bulk_events(
             router.bulk_send(self.bulk_transformed_events)
 
         overridden_events = self.bulk_transformed_events.copy()
+        overridden_events.pop()
```

Comment:
Can you comment why you're doing this? I'm sure I will forget as soon as this is merged. :)